packages/custom/orders/server/models/orders.js

```diff
@@ -0,0 +1,185 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var mongoose = require('mongoose'),
+  Schema = mongoose.Schema;
+
+
+/**
+ * Order Schema
+ */
+var OrderSchema = new Schema({
+  created: {
+    type: Date,
+    default: Date.now
+  },
+  user: {
+    type: Schema.ObjectId,
+    ref: 'User'
+  },
+  shipping: {
+    type: Number,
+    required: false,
+    trim: true
+  },
+  tax: {
+    type: Number,
+    required: false,
+    trim: true
+  },
+  taxRate: {
+    type: Number,
+    required: false,
+    trim: true
+  },
+  subTotal: {
+    type: Number,
+    required: false,
+    trim: true
+  },
+  totalCost: {
+    type: Number,
+    required: false,
+    trim: true
+  },
+
+  items:[
+    {
+      id: {
+        type: String,
+        required: false,
+        trim: true
+      },
+      name: {
+        type: String,
+        required: false,
+        trim: true
+      },
+      price: {
+        type: Number,
+        required: false,
+        trim: true
+      },
+      quantity: {
+        type: Number,
+        required: false,
+        trim: true
+      },
+      data:{
```

Comment:
Here you can define data as 
data: {
      type: Schema.Types.Mixed, 
      required : true
 }

Rather then listing all the fields.